sdk/documentintelligence/Azure.AI.DocumentIntelligence/tests/DocumentIntelligenceClient/DocumentClassifierLiveTests.cs

```diff
@@ -71,10 +71,10 @@ private void ValidateGenericClassifierResult(AnalyzeResult analyzeResult, string
         {
             Assert.That(analyzeResult.ModelId, Is.EqualTo(classifierId));
             Assert.That(analyzeResult.ApiVersion, Is.EqualTo(ServiceVersionString));
-            Assert.That(analyzeResult.StringIndexType, Is.EqualTo(StringIndexType.TextElements));
             Assert.That(analyzeResult.ContentFormat, Is.Not.EqualTo(default(DocumentContentFormat)));
 
-            Assert.That(analyzeResult.Content, Is.Empty);
+            // TODO: https://github.com/Azure/azure-sdk-for-net/issues/47482
+            //Assert.That(analyzeResult.Content, Is.Empty);
```

Comment:
Service issue. Tracking here so we don't forget to re-enable this assertion once it's fixed: https://github.com/Azure/azure-sdk-for-net/issues/47482